src/rpcq.lisp

```diff
@@ -43,6 +43,10 @@
 ;; store all messages defined thus far in their namespace
 (defvar *messages* (make-hash-table :test 'equal))
 
+;; Wrapper for messagepack's treatment of false/nil.
+(defvar *use-false* t
```

Comment:
is this going to break anything

Reply:
Perhaps, if someone in python-land is expecting to receive `None` rather than `False`. I think this is the right way to go though.